src/css/skins/seven.less

```diff
@@ -1,21 +1,20 @@
 @import "../imports/mixins";
 
 .jw-skin-seven {
-    @active-color: #FF0046;
-    @inactive-color: #fff;
+    @active-color: #ffffff;
+    @inactive-color: rgb(153, 153, 153);
     
     @cc-inactive: @inactive-color;
     @cc-active: @active-color;
 
     @rail-height: .2em;
 
     .jw-background-color {
-        background: #000;
+        background: rgba(37, 37, 37, 0.7); 
```

Comment:
@robwalch Why not add transparency? The control bar and elements shouldn't black out the video behind it.